flake8diff/exceptions.py

```diff
@@ -1,7 +1,12 @@
 from __future__ import unicode_literals, print_function
 
 
-class Flake8NotInstalledError(Exception):
+class BaseError(Exception):
```

Comment:
np. `six` is pretty awesome. another good lib is [`future`](http://python-future.org/). it overlaps with `six` but is more comprehensive. its docs are awesome and have lots of examples of differences between 2 and 3.